.github/workflows/internal-archive-release.yml

```diff
@@ -98,12 +101,13 @@ jobs:
       - name: "Checkout provided ref, default to branch main"
         uses: actions/checkout@v4
         with:
-          ref: ${{ inputs.ref || 'main' }}
+          repository: "${{ inputs.org }}/dbt-${{ inputs.dbms_name }}"
+          ref: "${{ inputs.ref || 'main' }}"
 
       - name: "Validate patch version input against patch version of ref"
         id: validate_version
         run: |
-          version_in_file="$(grep 'version =' "dbt/adapters/${{ inputs.dbms_name }}/__version__.py" | cut -d '"' -f2)"
+          version_in_file="$(grep -E 'version(: str)? =' "dbt/adapters/${{ inputs.dbms_name }}/__version__.py" | cut -d '"' -f2)"
```

Comment:
I think this is always true. The `hatch` setup tutorial uses `__about__.py` instead of `__version__.py` and we accounted for that in the adapter registration function. However, I think we hard coded this path elsewhere and reverted our adapters to leave the version in `__version__.py`.